### PR TITLE
Migrate to Vidarr

### PR DESCRIPTION
**Pull request by asofmosq**

### CODE REVIEW

**Comment by asofmosq** [comment]
retest this please

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .idea/
-*.iml
\ No newline at end of file
+*.imlv.out
```
Comment (apmasell) [inline .gitignore]: Hmm. Looks like this needed an extra newline.

**Comment by apmasell** [comment]
Why is the `compare.sh` empty? That's probably why it's succeeding. Try copying the compare script from https://github.com/oicr-gsi/pipedev/blob/master/pipedev-niassa-cromwell-workflow/src/test/resources/compare.sh

**Comment by asofmosq** [comment]
Not sure why the diff says it's an empty file https://github.com/oicr-gsi/mutect2/blob/migrate-to-vidarr/tests/compare.sh

**Review by apmasell — APPROVED** [review]

**Pull request merged**